sdks/js/packages/core/react/components/organization/plans/pricing-column.tsx

```diff
@@ -0,0 +1,354 @@
+import { useNavigate } from '@tanstack/react-router';
```

Comment:
[Nitpick] Should we try to set the linter to automatically sort imports alphabetically, and segment them according to whether they are external imports or internal ones?

Reply:
we can sort alphabetically, but sorting external and internal are difficult as we use alias and relative path for internal imports.